### PR TITLE
fix: stringify body functions.invoke

### PR DESCRIPTION
**Pull request by thorwebdev**
- https://github.com/supabase/supabase/issues/6267
- https://github.com/supabase/functions-js/issues/12

### CODE REVIEW

File: web/spec/gotrue.json [hunk 1]
```diff
@@ -11058,8 +11059,17 @@
 				}
 			],
 			"type": {
-				"type": "literal",
-				"value": "sms"
```
Comment (inian) [inline web/spec/gotrue.json:11062]: There are some gotrue type changes here. Is this intentional? Split it up into a different PR?

**Review by inian — APPROVED** [review]

File: web/spec/gotrue.json (hunk 1 above)
Comment (thorwebdev) [reply web/spec/gotrue.json:11062]: this us auto gened, so I think someone forgot to run `npm run gen:supabase` in a previous PR?
